.travis.yml

```diff
@@ -16,6 +16,7 @@ script:
   - npm run -s build
   - bower install
   - npm -s test
+  - npm run -s bench
```

Comment:
I'd rather not add this to the CI script, as I can imagine it causing CI to take a lot longer if we add more things to the benchmarking script.